executor/common.h

```diff
@@ -20,12 +20,32 @@
 #include <sys/wait.h>
 
 #include <linux/capability.h>
-#include <linux/if.h>
-#include <linux/if_tun.h>
 #include <linux/kvm.h>
 #include <linux/sched.h>
+
+#include <arpa/inet.h>
+#include <linux/if.h>
+#include <linux/if_ether.h>
+#include <linux/if_tun.h>
+#include <linux/ip.h>
+#include <linux/tcp.h>
 #include <net/if_arp.h>
 
+// Can't include <linux/ipv6.h>, since it causes
+// conflicts due to some structs redefinition.
+struct ipv6hdr {
```

Comment:
Move this closer to syz_extract_tcp_res. This needs to be under #ifdef __NR_syz_extract_tcp_res to not appear in reproducers when not needed.